man/tpm2_getekcertificate.1.md

```diff
@@ -2,52 +2,72 @@
 
 # NAME
 
-**tpm2_getekcertificate**(1) - Retrieve the Endorsement key Certificate for the
-TPM endorsement key from the TPM manufacturer's endorsement certificate hosting
-server.
+**tpm2_getekcertificate**(1) - Retrieve the Endorsement key Certificate.
 
 # SYNOPSIS
 
 **tpm2_getekcertificate** [*OPTIONS*] [*ARGUMENT*]
 
 # DESCRIPTION
 
-**tpm2_getekcertificate**(1) - Retrieve the Endorsement key Certificate for
-the TPM endorsement key from the TPM manufacturer's endorsement certificate
-hosting server. The argument specifies the URL address for the ek certificate
-portal.
+**tpm2_getekcertificate**(1) - Retrieve the endorsement key certificate. The
+certificate is present either on the TCG specified TPM NV indices OR on the TPM
+manufacturer's endorsement certificate hosting server. The certificate lookup
+is in the following order:
+1. On the TPM NV indices: if an EK public file is **not specified** and argument
+   specifying the URL address for the EK certificate portal is **not supplied**.
+2. On the Intel EK certificate backend: if an EK public file, either RSA or ECC,
```

Comment:
From the Intel EK Certificate web hosted backend under the following condition: ...